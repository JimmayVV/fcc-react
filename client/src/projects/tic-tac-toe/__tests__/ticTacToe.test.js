import React from "react";
import { render } from "react-testing-library";
import TicTacToe, { Heading } from "../index";
import GameBoard from "../gameBoard";

it("should have 1 heading and 1 game board", () => {
  const { container } = render(<TicTacToe />);
  expect(container.firstChild).toMatchSnapshot();
});
