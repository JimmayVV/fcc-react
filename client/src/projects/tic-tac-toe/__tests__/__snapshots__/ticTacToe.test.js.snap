// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have 1 heading and 1 game board 1`] = `
<div
  class="ticTacToe-container"
>
  Tic Tac Toe!
  <button
    type="button"
  >
    Reset
  </button>
  <div
    class="ticTacToe-board"
  >
    <div
      class="ticTacToe-cell"
      role="button"
      tabindex="-1"
    >
      <div
        class="ticTacToe-cell__content"
      />
    </div>
    <div
      class="ticTacToe-cell"
      role="button"
      tabindex="-1"
    >
      <div
        class="ticTacToe-cell__content"
      />
    </div>
    <div
      class="ticTacToe-cell"
      role="button"
      tabindex="-1"
    >
      <div
        class="ticTacToe-cell__content"
      />
    </div>
    <div
      class="ticTacToe-cell"
      role="button"
      tabindex="-1"
    >
      <div
        class="ticTacToe-cell__content"
      />
    </div>
    <div
      class="ticTacToe-cell"
      role="button"
      tabindex="-1"
    >
      <div
        class="ticTacToe-cell__content"
      />
    </div>
    <div
      class="ticTacToe-cell"
      role="button"
      tabindex="-1"
    >
      <div
        class="ticTacToe-cell__content"
      />
    </div>
    <div
      class="ticTacToe-cell"
      role="button"
      tabindex="-1"
    >
      <div
        class="ticTacToe-cell__content"
      />
    </div>
    <div
      class="ticTacToe-cell"
      role="button"
      tabindex="-1"
    >
      <div
        class="ticTacToe-cell__content"
      />
    </div>
    <div
      class="ticTacToe-cell"
      role="button"
      tabindex="-1"
    >
      <div
        class="ticTacToe-cell__content"
      />
    </div>
  </div>
</div>
`;
